scripts/package

```diff
@@ -62,10 +62,10 @@ if [ ! -d build/charts ]; then
     done
 
     # Remove index to force building a virtual index like system charts
-    rm -f $INDEX_PATH $CHART_REPO_DIR/assets/index.yaml
+    rm -f $INDEX_PATH $CHART_REPO_DIR/assets/index.yaml $CHART_REPO_DIR/released/assets/index.yaml
```

Comment:
Does the standard index.yaml get used, or does the image export create a new index.yaml and use that ?

Reply:
The image export uses `helm.LoadIndex()`, which will build an index if there is no `index.yaml` file.

Given that we were already removing the other `index.yaml` files, I assume this is the desired behavior.

Reply:
As long as that feature takes both folders into accounts and includes/sorts them correctly, then yes.

Reply:
I don't think the sorting works due to our uncommon semver format, but it shouldn't matter because when we prep it (extract the tarballs using index.yaml) it is expected that only the latest version of each chart is extracted and no sorting will be needed to find the latest. If that's still the case, then it should be ok. As a side note, this behavior will need to change once we add support for feature charts min/max in the generator because it was implemented this way to reuse the logic we already had in place for system-charts, but building on top of this will lead to complexity and bugs.